lib/fd_send_recv_stubs.c

```diff
@@ -64,7 +64,7 @@ CAMLprim value stub_unix_send_fd(value sock, value buff, value ofs, value len, v
 
   cfd = Int_val(fd);
 
-  cv_flags = convert_flag_list(flags,msg_flag_table);
+  cv_flags = caml_convert_flag_list(flags,msg_flag_table);
```

Comment:
Can't add a comment on that line specifically but below on line 102 there is a `perror` which might destroy errno, if we want to preserve it then errno should be saved beforehand. However calling `perror` is probably not nice for a library and we'd be better off by just deleting that line.

Reply:
perror removed